src/monsters/turael/Icefiend.ts

```diff
@@ -0,0 +1,25 @@
+import LootTable from '../../structures/LootTable';
+import SimpleMonster from '../../structures/SimpleMonster';
+import { GemTable } from '../subtables/RareDropTable';
+
+export const IcefiendTable = new LootTable(128)
```

Comment:
currently the total weights on the table is 37, this means youll get 1 drop every 3.4 kills. i'd change this lower to like ~50 or maybe even 37 to guarantee a drop every kill